node/node.go

```diff
@@ -388,7 +392,13 @@ func (nc *Config) Build(ctx context.Context) (*Node, error) {
 	//nwork := bsnet.NewFromIpfsHost(innerHost, router)
 	bswap := bitswap.New(ctx, nwork, bs)
 	bservice := bserv.New(bs, bswap)
-	fetcher := net.NewBitswapFetcher(ctx, bservice, blkValid)
```

Comment:
Once we are done with the next release and the graphsync fetcher has been tried by fire there will be very little reason to keep a bitswap fetcher around in the code.  Would you mind filing an issue for removing that code once the next release comes and goes?

Reply:
#3191